src/classes/PS_CommitmentRequest.cls

```diff
@@ -311,12 +367,21 @@ public inherited sharing class PS_CommitmentRequest {
                 return this;
             }
 
-            this.firstName = organization.Name;
-            this.lastName = organization.Name;
+            this.firstName = getMaxLenName(organization.Name);
+            this.lastName = getMaxLenName(organization.Name);
 
             return this;
         }
 
+        /**
+         * @description Return max number of character allowed for the first/last name in Elevate
+         * @param name First or Last Name
+         * @return String
+         */
+        private String getMaxLenName(String name) {
+            return String.isBlank(name) ? name : name.left(60);
```

Comment:
Probably best to put `60` into a constant at the top of the class just in case it changes in Elevate in the future.